templates/docs/examples/patterns/side-navigation/_default.html

```diff
@@ -1,12 +1,12 @@
-<div class="p-side-navigation {% if is_dark %}is-dark{% endif %}" id="drawer">
+<div class="p-side-navigation {% if is_dark %}is-dark{% endif %}" id="drawer" role="banner">
```

Comment:
Why is side navigation pattern a banner?

"A banner role represents general and informative content frequently placed at the beginning of the page. This usually includes a logo, company name, search icon, photo related to the page, or slogan."

Side navigation doesn't include logo, company name, search, slogan. It's navigation, nothing more nothing less. I don't feel like "banner" applies here at all.

Reply:
I think it applie in the case of a sidenav that serves as the main nav, as in the application layout.
I find this [MDN definition](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_role#:~:text=A%20banner%20landmark%20role%20converts,is%20applied%20into%20a%20header.&text=Assistive%20technologies%20can%20identify%20the,main%20%2C%20nav%20or%20section%20subsection.) more instructive. 

In the case of the docs sidenav it doesn't apply, indeed. I wil lremove it from there. Do you think we can keep it in the application sidenav example?

Reply:
In the application layout (full layout example) we used banner landmark (via `<header>`) on `l-navigation` - whole navigation region.
Side navigation component is just one part of this region (which also contains logo, etc).
So yes, banner landmark should be in application layout navigation region, but not on side navigation component (which by itself is not a banner).

Reply:
I think Ant was right here, that components should not have landmarks themselves. Landmarks should come from layout containers.

Possibly the only exception is the main navigation. Which by itself is kind of a layout type component and can have banner role.
Other patterns should not define landmarks.